packages/dagre-reactjs/src/layout-dagre.ts

```diff
@@ -43,13 +44,17 @@ export class LayoutDagre extends Layout implements GraphLayout {
     );
 
     this.graph.nodes().forEach((n) => this.graph.removeNode(n));
-    this.graph.edges().forEach((e) => this.graph.removeEdge(e.v, e.w));
+    this.graph.edges().forEach((e) => {
+      // Typescript compiler complains that removeEdge only expects 2 args
+      // @ts-ignore
+      this.graph.removeEdge(e.v, e.w, e.name);
```

Comment:
We should add a custom .d.ts extension file to add just this method signature to the Graph class.  It's currently missing from the types definition on definitelytyped and I don't know if they are being maintained.  If they were being maintained the final solution would be to get the correct type definition added over there but for now we could use definition merging to add the function.

Current definitions
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dagre/index.d.ts

I think this is probably because graphlibs api doc is out of date with the code https://github.com/dagrejs/graphlib/wiki/API-Reference#removeEdge
https://github.com/dagrejs/graphlib/blob/c464a1c9e60766aa93b491ee584e9a72768ba3d7/lib/graph.ts#L601